health/notifications/alarm-notify.sh.in

```diff
@@ -344,9 +336,7 @@ PD_SERVICE_KEY=
 FLEEP_SENDER="${host}"
 
 # Amazon SNS configs
-DEFAULT_RECIPIENT_AWSSNS=
 AWSSNS_MESSAGE_FORMAT=
-declare -A role_recipients_awssns=()
```

Comment:
`role_recipients_awssns` is removed because it is unused. It is only declared.

Reply:
if you search for `role_recipients`, you'll see how this is supposed to work. There's a part of the code that iterates through `method_names` to build the `to_` variables that are otherwise left unset. This one didn't make it there for some reason. I'll see if I can set up these notifications to do some tests, because I don't see how they can be working even before this PR.

Reply:
Ok, verified that it doesn't work even before the PR. The deletions you've done are fine, but please add a line 165 with `awssns`, so that `method_names` includes it. This will fix the notifications.

Reply:
So it seems that no one is using this notification method as it doesn't work and there was no bug reports regarding it. Looks like a dead and unmaintained feature.

Reply:
fixed

Reply:
@cakrit This appears to have been an oversight on my part when resolving the merge conflict involving the AWS SNS support and the PR that redid the recipient finding logic.  Assuming that it gets added to `method_names`, it should work as intended.

@paulfantom When I originally wrote this, I actually did plan to use it, I just never ended up actually using it.  I would argue it's probably worth keeping though because providing support for big cloud providers' services makes us look good (provided they work of course).